var/spack/repos/builtin/packages/mold/package.py

```diff
@@ -12,8 +12,6 @@ class Mold(CMakePackage):
     homepage = "https://github.com/rui314/mold"
     url = "https://github.com/rui314/mold/archive/refs/tags/v1.11.0.tar.gz"
 
-    maintainers("jabcross")
-
```

Comment:
@tldahlgren feel free to add me as a maintainer. I can at least commit to adding new releases for the near future; more complicated stuff I probably don't have the knowledge for (I'm only an occasional, casual user of mold) but I'll have a look if something comes up.

I can also add myself in https://github.com/spack/spack/pull/44034 if you want to close this without merging?

Reply:
Looks like that was merged so let me add you.